script/install-bundler

```diff
@@ -16,10 +16,10 @@ set -e
 
 # `grep`:      find the occurrences of "BUNDLED WITH" (-m is unnecessary and behaves different on macOS)
 # `-A`:        print the next line after "BUNDLED WITH" as well
-# `-x -F`:     find exactly that string without interpreting regex
+# `-F`:        find exactly that string
 # `tail -n 1`: take the last line, the version line
 # `tr -d`:     delete all spaces, the indent before the version
-version="$(grep -A 1 -x -F "BUNDLED WITH" Gemfile.lock | tail -n 1  | tr -d '[:space:]')"
+version="$(grep -A 1 -F "BUNDLED WITH" Gemfile.lock | tail -n 1  | tr -d '[:space:]')"
```

Comment:
I think that we can actually remove the whole version parsing because modern bundler can do this automatically. But I'm only 80% sure about this.

Reply:
> I think that we can actually remove the whole version parsing because modern bundler can do this automatically. But I'm only 80% sure about this.

I thought so too, and the documentation says that Bundler will automatically switch based on this version number: https://bundler.io/guides/bundler_2_upgrade.html#version-autoswitch

But it only seems to look at the major version number:
```
$ grep -A 1 "BUNDLED WITH" Gemfile.lock
BUNDLED WITH
   2.4.2

$ bundle version
Bundler version 2.4.3 (2023-01-06 commit ed1f27f75c)
```

So I guess we do still need it.